packages/opencensus-core/src/trace/model/no-record/no-record-span.ts

```diff
@@ -184,25 +184,12 @@ export class NoRecordSpan implements types.Span {
 
   /**
    * Starts a new no record child span in the no record root span.
-   * @param nameOrOptions Span name string or SpanOptions object.
-   * @param kind Span kind if not using SpanOptions object.
+   * @param [options] A SpanOptions object to start a child span.
```

Comment:
What's the role of the brackets [ ] around `options` in the JSDoc? Does that signify that it's optional?

Reply:
Yes, brackets are used to indicate optional param.